### PR TITLE
feat: 🎸 Add 3 new shortcut popup icons

### PR DESCRIPTION
**Pull request by kamack38**
Signed-off-by: kamack38 <64226248+kamack38@users.noreply.github.com>

### CODE REVIEW

File: lib/awful/hotkeys_popup/widget.lua [hunk 1]
```diff
@@ -169,9 +169,9 @@ widget.labels = {
     -- Basic multimedia keys:
     XF86MonBrightnessUp   = "🔆+",
     XF86MonBrightnessDown = "🔅-",
-    XF86AudioRaiseVolume = "Vol+",
-    XF86AudioLowerVolume = "Vol-",
-    XF86AudioMute = "Mute",
+    XF86AudioRaiseVolume = "🕩+",
+    XF86AudioLowerVolume = "🕩-",
+    XF86AudioMute = "⏹",
```
Comment (Aire-One) [inline lib/awful/hotkeys_popup/widget.lua:174]: I don't think the "Stop Button" is appropriate for the XF86AudioMute key.

According to https://emojipedia.org/muted-speaker/, the "Muted Speaker" is part of Unicode 6.0, which is the same Unicode Standard version that introduced the brightness symbols used by XF86MonBrightnessUp and XF86MonBrightnessDown. So I guess it's ok to use it.

**Review by Aire-One — CHANGES_REQUESTED** [review]

File: lib/awful/hotkeys_popup/widget.lua (hunk 1 above)
Comment (kamack38) [reply lib/awful/hotkeys_popup/widget.lua]: Fixed in the newest commit

**Review by Aire-One — APPROVED** [review]
Thank you!

**Review by sclu1034 — APPROVED** [review]

**Review by actionless — APPROVED** [review]
thanks 😸👍

**Pull request merged**